examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Express examples
+
+This page contains list of examples using Express.
+
+- Authentication with login and passowrd - [examples/auth](./auth)
+- HTTP content negotiation - [examples/content-negotiation](./content-negotiation)
+- Working with cookie-based session - [examples/cookie-sessions](./cookie-sessions)
+- Working with cookies via [`cookie-parser`](https://github.com/expressjs/cookie-parser) - [examples/cookies](./cookies)
+- Files transfer to client - [examples/downloads](./downloads)
```

Comment:
The description of this one sounds strange for some reason reading it. I think because the plural is in the wrong place. Is the "s" a typo?

Reply:
Changed to `Transferring files to client`